README.md

```diff
@@ -20,6 +20,41 @@ To sign in to Chartio:
 - Open the [FT Chartio](https://chartio.com/financialtimes/) in your browser.
 - Press the "Sign in with Google" button and select your FT Google Account.
 
+### How to track extra click data using `data-trackable` attributes
+
+You can add extra information to o-tracking events by using the `data-trackable` and `data-trackable-context-*` custom attributes. 
+
+
+#### `data-trackable`
+
+`o-tracking` will walk up the DOM to get the tree of elements with `data-trackable` attributes. For example, clicks to the list-item button below will result in an event that has this path recorded within it - `header-menu country-selector china`:-
+
+```html
+<div data-trackable="header-menu">
+	<span>
+		<div data-trackable="country-selector">
+			<ul>
+				<li data-trackable="china" role="button">China</li>
```

Comment:
Could we update this example to use a `button` element instead? [`role="button"` would need more work to make it focusable and respond to enter/space etc](https://www.w3.org/TR/wai-aria-practices/examples/button/button.html)

Reply:
```suggestion
				<li data-trackable="china"><button>China</button></li>
```